core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/ProcessUtil.java

```diff
@@ -66,7 +68,7 @@ public static void process(Process p, LineConsumer stdout, LineConsumer stderr)
    * @param stdout a consumer where stdout will be redirected
    * @throws IOException
    */
```

Comment:
-1 to change method name. It can be used by someone else.

Reply:
Agree